tests/tasks/docker/test_images.py

```diff
@@ -105,6 +107,81 @@ def test_repository_is_replaced(self, monkeypatch):
         task.run(repository="test")
         assert api.return_value.pull.call_args[1]["repository"] == "test"
 
+    def test_logs_docker_stream_values(self, monkeypatch, caplog):
+
+        task = PullImage(repository="original")
+
+        api = MagicMock()
+        docker_output = ["first message", "second message", "third message"]
```

Comment:
Placeholder fake data. Going to grab the real logs from downloading busybox and use here instead.

Reply:
You're probably going to want to keep it mocked with fake data. Unit tests which actually pull images aren't very portable and I know CircleCI throws a fit.